tools/pipelines/templates/include-test-real-service.yml

```diff
@@ -341,8 +347,9 @@ jobs:
       - task: Npm@1
         displayName: '[test] ${{ parameters.testCommand }} ${{ variant.flags }}'
         continueOnError: ${{ parameters.continueOnError }}
-        env:
-          ${{ parameters.env }}
+        ${{ if parameters.env }}:
+          env:
+            ${{ parameters.env }}
```

Comment:
The new stage doesn't use it right now, and if nothing is passed this ends up being a syntax error with the current code.